tests/unit/local/events/test_api_event.py

```diff
@@ -551,7 +551,6 @@ def test_to_dict_with_defaults(self):
             "rawQueryString": None,
             "cookies": None,
             "headers": None,
-            "queryStringParameters": None,
```

Comment:
Can we add a test case with the query string parameters present, since that is also a valid code path?

Reply:
I believe the test case above this one has query string parameters present

Reply:
Ok - let me look at the context instead of just the diff :P